cime_config/testdefs/testlist_mom.xml

```diff
@@ -58,15 +40,15 @@
       <option name="wallclock">01:30:00</option>
     </options>
   </test>
-  <test name="SMS" grid="T62_t025" compset="CMOM">
+  <test name="SMS" grid="T62_t025" compset="C">
```

Comment:
Are you still supporting the t025 grid in MOM here? Also, still supporting the older CORE forcing?